codex/sales/reservations.nim

```diff
@@ -359,6 +359,56 @@ proc createReservation*(
 
   return success(reservation)
 
+proc returnBytes*(
```

Comment:
Can we think of a more descriptive name for this function? It's a bit hard to know what it does outside of the context of this PR. Maybe `returnBytesToRepo()` or `cancelReservation()`?